tests/test_simple_put.cpp

```diff
@@ -242,7 +323,13 @@ int main(void) {
     cmocka_unit_test_teardown(test_simple_put_gzip, teardown),
     cmocka_unit_test_teardown(test_simple_put_zero_byte, teardown),
     cmocka_unit_test_teardown(test_simple_put_one_byte, teardown),
-    cmocka_unit_test_teardown(test_simple_put_skip, teardown)
+    cmocka_unit_test_teardown(test_simple_put_skip, teardown),
+///#ifdef PUT_GET_LARGE_DATASET_TEST
+    cmocka_unit_test_teardown(test_simple_put_skip, donothing),
+    cmocka_unit_test_teardown(test_simple_get, teardown),
+    cmocka_unit_test_teardown(test_large_put_auto_compress, donothing),
+    cmocka_unit_test_teardown(test_large_get, teardown)
+///#endif
```

Comment:
Large put and large get will be conditionally enabled with a MACRO and will not always run.